exercises/transpose/transpose_test.go

```diff
@@ -0,0 +1,41 @@
+package transpose
+
+import (
+	"fmt"
+	"math"
+	"testing"
+)
+
+// For a more visual representaion of the test data than is provided by
+// cases_test.go have a look at the original json data here:
+// https://github.com/exercism/x-common/blob/master/exercises/transpose/canonical-data.json
+
+func TestTranspose(t *testing.T) {
+	for _, test := range testCases {
+		actual := Transpose(test.input)
+		if fmt.Sprintf("%s", actual) != fmt.Sprintf("%s", test.expected) {
+			// find the shorter of actual and expected, to avoid index out of range errors
+			min := int(math.Min(float64(len(test.expected)), float64(len(actual))))
+			// let's get the line the error is on, so we avoid a wall of text to debug
+			for i := 0; i < min; i++ {
+				if test.expected[i] != actual[i] {
+					t.Fatalf("\n\tTranspose(%q): %s\n\n\tExpected: %q\n\tGot: %q\n\n\tRow %d Expected: %q Got: %q",
+						test.input, test.description, test.expected, actual, i, test.expected[i], actual[i])
+				}
+			}
+		}
+	}
+}
+
+func BenchmarkTranspose(b *testing.B) {
+	b.StopTimer()
+	for _, test := range testCases {
+		b.StartTimer()
```

Comment:
We might like to do the same as #453 and remove these, since there isn't really any initialisation code required (the usual reason for starting/stopping timer). I suppose one could consider that iterating over the `testCases` is something, but I don't think it has such a large impact. I wish I had examples so that I could know whether I'm right on that.